trajopt_sco/include/trajopt_sco/osqp_interface.hpp

```diff
@@ -64,8 +66,13 @@ class OSQPModel : public Model
 public:
   OSQPModel();
   ~OSQPModel() override;
-  OSQPModel(const OSQPModel&) = default;
-  OSQPModel& operator=(const OSQPModel&) = default;
+  OSQPModel(const OSQPModel& model) : Model(model), P_(new csc{}), A_(new csc{}) {}
```

Comment:
I recommend moving implementation to cpp. Also does setting this to nullptr not work?

Reply:
Sure. I also fixed my assignment operator. Would you rather me delete that as well?